mirny.py

```diff
@@ -321,13 +321,17 @@ def __init__(self, platform):
             tsi = TSTriple()
             self.specials += tsi.get_tristate(m)
             self.comb += [
-                tsi.o.eq(regs[3].write[i] | (0 if i >= 4 else
-                    (regs[1].write[11] & eem[i + 4].i))),
+                tsi.o.eq(regs[3].write[i]),
                 regs[3].read[i].eq(tsi.i),
                 tsi.oe.eq(regs[3].write[i + 8]),
                 regs[3].read[i + 8].eq(tsi.oe),
             ]
 
+        # set remaining Almazny shift register NOE to 0 at all times
+        for pin in platform.request("almazny_noe"):
+            x = Signal()
+            self.comb += [ x.eq(0), pin.eq(x) ]
```

Comment:
just `pin.eq(0)`

Reply:
It can't work with ``pin.eq(0)``, there is an error ``ERROR:HDLCompilers:168 - "mirny.v" line 313 No signals referenced in statement with implicit sensitivity list``.